README.markdown

```diff
@@ -102,14 +104,29 @@ Descriptive and consistent naming makes software easier to read and understand.
 - prefer methods and properties to free (or global) functions.
 - case for acronyms and initialisms should be uniformly up or down
 - use the same base name for methods that share the same meaning
-- avoid overloads based purely on return type
+- avoid overloads based purely on return type. Avoid using [Covariant Returns](https://en.wikipedia.org/wiki/Covariant_return_type) which are allowed in Swift.
 - choose good parameter names that serve as documentation
 - label closure and tuple parameters
+  - e.g.:
+
+Instead of:
+```swift
+func makeLocation() -> (Float, Float) {
+   // ... 
+}
+```
+do:
+```swift
+func makeLocation() -> (latitude: Float, longitude: Float) {
```

Comment:
We mainly go with an external and internal parameter name within the projects app `func makeLocation() -> (from latitude: Float, to longitude: Float) {` Should this be mentioned in this section? Or do people disagree with this style? Or should it just be up to the individual to decided?

Reply:
I'm not against that. Just adding an example to get a clearer idea. Exactly to have this kind of discussion 😄